main/main.cpp

```diff
@@ -575,7 +575,10 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("", "--fixed-fps is forced when enabled, but it can be used to change movie FPS.\n");
 	print_help_option("", "--disable-vsync can speed up movie writing but makes interaction more difficult.\n");
 	print_help_option("", "--quit-after can be used to specify the number of frames to write.\n");
-
+#ifdef WINDOWS_ENABLED
```

Comment:
```suggestion
#if defined(WINDOWS_ENABLED) && !defined(WINDOWS_SUBSYSTEM_CONSOLE)
```

If it's compiled as a console executable, spawning another one is redundant.